Project flow/Certificates, provisioning profiles and APNS.md

```diff
@@ -68,9 +70,26 @@ Ad hoc profiles are used to distribute the app to the devices which are chosen w
 
 TestFlight is another common way of sending the builds directly to clients. You use the App Store provisioning profile type, and the build is uploaded to App Store directly.
 
-## APNS
+## APNS - p8 key
+
+For the iOS platform, push notifications are available through the APNS server. In order to establish communication with the APNS server, you'll have to create the Apple Push Notification service key. Once that key is created, you should upload it to the push notification provider (e.g. Firebase) configuration page or to the server if notification handling is done manually.
+
+To create the key, visit the [developer console](https://developer.apple.com) and select the *Certificates, Identifiers & Profiles* option. On the next page, select *Keys* and then *+*. Add the key name (e.g. APNs key) and select *Apple Push Notification service (APNs)*.
+![ios_register_new_key](/img/ios_register_new_key.png)
```

Comment:
Currently, this image is not shown in the rich preview of the document, but referencing looks good.